src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -32,4 +53,29 @@ object IodbDataSource {
   private def storageVersionGen(): Array[Byte] = {
     ByteBuffer.allocate(java.lang.Long.SIZE / java.lang.Byte.SIZE).putLong(updateCounter.incrementAndGet()).array()
   }
+
+  def apply(path: String, keySize: Int): IodbDataSource = {
+    //Delete directory if it existed and create a new one
+    val dir: File = new File(path)
+    val dirCreationSuccess = (!dir.exists() || deleteDirectory(dir)) && dir.mkdir()
+    assert(dirCreationSuccess, "Iodb folder creation failed")
+
+    val lSMStore: LSMStore = new LSMStore(dir = dir, keySize = keySize, keepSingleVersion = true)
```

Comment:
You are right, I changed the way in which the `clear` is implemented in the most recent commit.